src/renderer/tvgText.cpp

```diff
@@ -71,6 +71,22 @@ Result Text::load(const std::string& path) noexcept
 }
 
 
+Result Text::load(const char* name, const char* data, uint32_t size, const string& mimeType, bool copy) noexcept
+{
+    if (!name) return Result::InvalidArguments;
+
+    //unload font
+    if (!data && size == 0) {
+        if (LoaderMgr::retrieve(name)) return Result::Success;
+        return Result::InvalidArguments;
+
+    } else if (!data || size <= 0) return Result::InvalidArguments;
```

Comment:
Why not do an unload in this case?

Reply:
to prevent unloading by a mistake. to distinguish between wrong args and unloading. like with a dash api - if the user really wants to unload, has to pay attention and pass null and 0.

Reply:
@JSUYA it is changed now